content-scripts/src/index.js

```diff
@@ -62,6 +61,8 @@ const init = async () => {
   changeCourseCards();
   fixPagination();
   changeLayout();
+  
+  rememberLogin(data);
```

Comment:
Any reason to be the last function executing?

Reply:
It was because of the layout

Reply:
Your PR should fix this